extensionsGallery.json

```diff
@@ -3233,7 +3233,7 @@
 								},
 								{
 									"key": "Microsoft.AzDataEngine",
-									"value": ">=1.26.0"
+									"value": ">=1.27.0"
```

Comment:
We haven't released 1.27 yet so this isn't a valid engine for the stable gallery. Remove the update to that file from this PR and then submit a new PR against the [extensions/RC1](https://github.com/microsoft/azuredatastudio/tree/extensions/rc1) branch for the stable gallery update as per https://github.com/microsoft/azuredatastudio-docs/blob/main/Guides/UpdateExtensionGallery.md#scenarios